ext/ffi/jit_trampoline.rs

```diff
@@ -58,12 +58,23 @@ fn native_to_c(ty: &NativeType) -> &'static str {
 
 pub(crate) fn codegen(sym: &crate::Symbol) -> String {
   let mut c = String::from(include_str!("prelude.h"));
-  let ret = native_to_c(&sym.result_type);
+  let needs_unwrap = matches!(
+    &sym.result_type,
+    NativeType::Function
+      | NativeType::Pointer
+      | NativeType::I64
+      | NativeType::ISize
+      | NativeType::U64
+      | NativeType::USize
+  );
```

Comment:
Seems like the exact same expression is in ffi/lib.rs?
```suggestion
  let needs_unwrap = crate::needs_unwrap(sym.result_type);
```